src/app/item-popup/PlugTooltip.tsx

```diff
@@ -70,8 +70,9 @@ export default function PlugTooltip({
       {defs && Boolean(plug?.plugItem?.investmentStats?.length) && (
         <div className="plug-stats">
           {plug.stats &&
-            _.sortBy(Object.keys(plug.stats), (h) =>
-              statWhiteList.indexOf(parseInt(h, 10))
+            _.sortBy(
+              Object.keys(plug.stats),
+              (h) => statWhiteListObj[parseInt(h, 10)]
```

Comment:
If we found that this were the performance bottleneck (and it most certainly isn't), this is where a Map keyed on strings would be great, to avoid the `parseInt`, which is quite a bit more expensive than the lookup.

However, this is part of the render function of a tooltip and it's only going to get run every so often (when you hover the tooltip) and it operates on a very small list anyway. This is the sort of thing that I wouldn't bother changing.